src/lib/ProductExporter.ts

```diff
@@ -50,7 +50,7 @@ export class ProductExporter {
     }
 
     public async all(conn: core.Connection) {     
-        Util.removeDir(this.dir);    
+        // Util.removeDir(this.dir);    <- ziom nie usuwaj katalogu
```

Comment:
@adrianlukaszek1992 przy eksporcie nie powinniśmy usuwać katalogu który już jest na dysku
eksportując kolejny raz to samo chcę widzieć diff'a w gicie